extensions/sql-database-projects/src/models/project.ts

```diff
@@ -555,33 +548,14 @@ export class Project implements ISqlProject {
 			this._databaseReferences.push(new SystemDatabaseReferenceProjectEntry(
 				Uri.file(''),
 				Uri.file(''), // TODO: remove these after add and delete are swapped - DacFx handles adding and removing system dacpacs, so we don't need to keep track of the paths here
-				systemDbReference.systemDb === mssql.SystemDatabase.master ? constants.master : constants.msdb,
+				systemDbReference.systemDb === mssql.SystemDatabase.Master ? constants.master : constants.msdb,
```

Comment:
Addressed completely in https://github.com/microsoft/azuredatastudio/pull/22196, but this change was necessary here for this to build.